internal/v5/archive_test.go

```diff
@@ -491,6 +491,26 @@ func (s *ArchiveSuite) TestPutCharmWithInvalidChannel(c *gc.C) {
 	)
 }
 
+func (s *ArchiveSuite) TestPutMultiseriesCharm(c *gc.C) {
+	s.assertUploadCharm(c, "PUT", newResolvedURL("~charmers/juju-gui-2", -1), "multi-series", nil)
+}
+
+func (s *ArchiveSuite) TestPutBundle(c *gc.C) {
+	// Upload the required charms.
+	for _, rurl := range []*router.ResolvedURL{
+		newResolvedURL("cs:~charmers/utopic/mysql-42", 42),
+		newResolvedURL("cs:~charmers/utopic/wordpress-47", 47),
+		newResolvedURL("cs:~charmers/utopic/logging-1", 1),
+	} {
+		err := s.store.AddCharmWithArchive(rurl, storetesting.Charms.CharmArchive(c.MkDir(), rurl.URL.Name))
+		c.Assert(err, gc.Equals, nil)
+		err = s.store.Publish(rurl, nil, params.StableChannel)
+		c.Assert(err, gc.Equals, nil)
+	}
+
+	s.assertUploadBundle(c, "PUT", newResolvedURL("~charmers/bundle/wordpress-simple-2", -1), "wordpress-simple")
+}
+
```

Comment:
Please could you add a PUT equivalent of TestPostSingleSeriesCharmWhenMultiSeriesVersionExists just to make sure that isn't possible.

Reply:
done